assets/js/components/Work/Tabs/About.jsx

```diff
@@ -160,10 +162,12 @@ const WorkTabsAbout = ({ work }) => {
         relatedUrl,
         relatedMaterial,
         rightsHolder,
-        rightsStatement: {
-          id: data.rightsStatement,
-          scheme: "RIGHTS_STATEMENT",
-        },
+        rightsStatement: data.rightsStatement
+          ? {
+              id: data.rightsStatement,
+              scheme: "RIGHTS_STATEMENT",
+            }
+          : {},
```

Comment:
So it's easier for the API/backend to handle an empty object as opposed to `null`?  If so, works for us.

Reply:
Well, it could do either. I wasn't sure which was the best or most consistent honestly. But I got two votes for doing it the empty object way yesterday in Slack...